defaults/main.yml

```diff
@@ -1,7 +1,17 @@
 ---
 agent_type: ""
-package_state: "present"
+package_state: present
 
-# default vars for the monitoring agent
+# Default vars for the monitoring agent
 monitoring_package_name: stackdriver-agent
 monitoring_service_name: stackdriver-agent
+
+# Local path to the config file to copy onto the remote server. Can be absolute or relative.
+monitoring_config_file_local: ""
+monitoring_config_file_remote: /etc/stackdriver/test.conf
+
+# Local path to the plugin dir whose contents will be copied onto the remote server. Can be absolute or relative.
+monitoring_plugin_dir_local: ""
+monitoring_plugin_dir_remote: /opt/stackdriver/collectd/etc/collectd.d/
```

Comment:
From collectd repo https://github.com/Stackdriver/agent-packaging/blame/f44e9002a0f17a54c2b82f70ea20cbbf97c06a17/collectd.conf#L130:

```
# if you have other config, especially for plugins, you can drop them
# into this directory
Include "/opt/stackdriver/collectd/etc/collectd.d"
Include "/etc/stackdriver/collectd.d"
```

Reading through https://github.com/Stackdriver/agent-packaging/pull/26 and b/72050498, i think we should use `/etc/stackdriver/collectd.d` instead of `/opt/stackdriver/collectd/etc/collectd.d` in the Ansible role.

Reply:
Ok, should we hardcode this as well?

Reply:
Yeah